summit/2018.html

```diff
@@ -54,20 +54,36 @@ <h2 class="text">Schedule</h2>
         </div>
       </div>
 
-      <div class="row">
+    <div class="row">
       <div class="col-md-10 col-md-offset-1">
         <img class="sponsor" src="/images/summit/megaphone.svg" width="65px" height="auto" alt="Sponsor"/>
-        <h2 class="text">Supporters</h2>
+        <h2 class="text">Sponsors</h2>
        <a href="https://stickermule.com/" target="_blank" rel="noopener"><img class="sponsor" src="/images/summit/stickermule-logo.svg" width="300px" height="auto" alt="Stickermule"/>
         <p class="data">Stickermule</p></a>
         <a href="https://mozilla.org/" target="_blank" rel="noopener"><img class="sponsor" src="/images/summit/mozilla-logo.svg" width="250px" height="auto" alt="Mozilla"/>
         <p class="data">Mozilla</p></a>
-        <a href="https://opentech.fund/" target="_blank" rel="noopener"><img class="sponsor" src="/images/summit/opentechfund-logo.svg" width="250px" height="auto" alt="OpenTechFund"/>
-        <p class="data">OpenTechFund</p></a>
         <a href="https://phplist.org/" target="_blank" rel="noopener"><img class="sponsor" src="/images/summit/phplist-logo.svg" width="220px" height="auto" alt="phpList"/>
         <p class="data">phpList</p></a>
       </div>
     </div>
+
+    <div class="row">
+      <div class="col-md-10 col-md-offset-1">
+        <img class="supporter" src="/images/summit/supporters.svg" width="65px" height="auto" alt="Sponsor"/>
+        <h2 class="text">Supporters</h2>
+        <a href="https://opentech.fund/" target="_blank" rel="noopener"><img class="supporter" src="/images/summit/opentechfund-logo.svg" width="180px" height="auto" alt="OpenTechFund"/>
+        <p class="data">OpenTechFund</p></a>
+        <a href="https://ura.design/" target="_blank" rel="noopener"><img class="supporter" src="/images/summit/ura-logo.svg" width="180px" height="auto" alt="Ura Design"/>
+        <p class="data">Ura Design</p></a>
+        <a href="https://redhat.com/" target="_blank" rel="noopener"><img class="supporter" src="/images/summit/redhat-logo.svg" width="200px" height="auto" alt="Red Hat"/>
+        <p class="data">Red Hat</p></a>
+        <a href="https://nextcloud.com/" target="_blank" rel="noopener"><img class="supporter" src="/images/summit/nextcloud-logo.svg" width="150px" height="auto" alt="Nextcloud"/>
+        <p class="data">Nextcloud</p></a>
+        <a href="https://creativecommons.org/" target="_blank" rel="noopener"><img class="supporter" src="/images/summit/cc-logo.svg" width="180px" height="auto" alt="Creative Commons"/>
```

Comment:
Having Creative Commons listed as supporter is a bit far fetched :) with this logic we should also list GitHub that made possible applying PRs for the website and W3C for creating HTML :)

Reply:
Creative Commons Fellowship program covered @cameralibre travel afaik

Reply:
Okay then, let's keep only Mozilla then (which sponsored financially anyway) @AnXh3L0